src/components/harp-face-fragment/harp-face-fragment-styles.ts

```diff
@@ -0,0 +1,36 @@
+import type { ViewStyle } from 'react-native'
+import { StyleSheet } from 'react-native'
+import type { HarpStrata } from 'harpstrata'
+
+import { getHarpFaceFacts } from '../../utils'
+import { columnWidth, rowHeight } from '../../styles'
+
+import { getFragmentFacts } from './utils'
+
+type Props = {
```

Comment:
Should rename all the Props to have more meaningful names.

I've realised that having all of the props types named Props means that some of the type warnings can be confusing.

Reply:
Done